sdk/security/arm-security/README.md

```diff
@@ -1,93 +1,110 @@
 ## Azure SecurityCenter SDK for JavaScript
 
-This package contains an isomorphic SDK for SecurityCenter.
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for SecurityCenter.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
```

Comment:
We are releasing an update to the code generator today that will result in the below changes. Suggesting this here so that you don't have to re-generate

```suggestion
- [LTS versions of Node.js](https://nodejs.org/about/releases/)
- Latest versions of Safari, Chrome, Edge, and Firefox.
```